Extensions/PrimitiveDrawing/shapepainterruntimeobject.ts

```diff
@@ -337,6 +345,26 @@ namespace gdjs {
       return this._clearBetweenFrames;
     }
 
+    setAntialiasingOn(value: boolean): void {
+      this._antialiasing = value;
+    }
+    
+    isAntialiasingOn(): boolean {
+      return this._antialiasing;
+    }
+
+    setAntialiasingQuality(value: integer): void {
+      if(value !== 2 && value !== 4 && value !== 8) {
```

Comment:
I think it could be argued that the fact we only accept 2, 4 or 8 as quality is an implementation detail from the renderer and should thereby only be enforced by the pixi renderer, not here. For all we know, a future renderer could accept only 4, 6 and 16. The RuntimeObject is not responsible for knowing the values accepted by each renderer and for only accepting one of those, it's up to the renderers to decide what values it accepts from the object and provide a default if the requested value is not possible.